SETUP.md

```diff
@@ -20,12 +20,9 @@ Launch your virtualization software and find an **Import** or **Import
 Appliance** option in the **File** menu. If you cannot locate an **Import**
 option, refer to your virtualization software's documentation.
 
-*Before* starting the VM for the first time, adjust its settings.  For best
-performance, allocate 4GB of memory. The default allocation of 3GB, but you
-might encounter stability and performance issues. Set the **Network Adapter**
-to **Bridged**.  Use an autodetect setting if available, or accept the default
```

Comment:
This is covered later in the networking configuration section--I'd just neglected to pull it out earlier